analyzer/src/main/kotlin/managers/GradleDependencyHandler.kt

```diff
@@ -86,7 +86,11 @@ class GradleDependencyHandler(
     override fun linkageFor(dependency: Dependency): PackageLinkage =
         if (dependency.isProjectDependency()) PackageLinkage.PROJECT_DYNAMIC else PackageLinkage.DYNAMIC
 
-    override fun createPackage(identifier: String, dependency: Dependency, issues: MutableList<OrtIssue>): Package? {
+    override fun createPackage(
```

Comment:
While you are at it, I think the `identifier` parameter can be completely dropped from `createPackage()`. It is not necessary, as the whole information is available via the `dependency` object. `GradleDependencyHandler` is currently the only implementation that accesses this parameter, but only for logging purposes; this could be solved differently.

Reply:
I saw this but wasn't sure...thanks for the hint! I'll file a follow up PR this afternoon.

Reply:
@oheger-bosch I've addressed it with #4037.